cluster/examples/kubernetes/ceph/object-openshift.yaml

```
@@ -95,3 +95,14 @@ spec:
     # Configure the pod liveness probe for the rgw daemon
     livenessProbe:
       disabled: false
+
+---
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  name: rook-ceph-rgw-my-store
+  namespace: rook-ceph
```

We need this comment to make it easier to change the namespace
```suggestion
  namespace: rook-ceph # namespace:cluster
```

---

Done.